### DISCUSSION TITLE
Getting powerful ships is too easy (quick)

### DISCUSSION THREAD

**Discussion by brendanjones** [endless-sky/endless-sky#9949] [Balance, Mechanics, and Interactions]
I was curious how long it'd take to get a Falcon, so I loaded up a new pilot and in real-time it took me 1 hour flat to have a Falcon with a fleet of 15 or so Bastions and Modified Argosies loaded with Heavy Lasers. That's the most powerful civilian ship available (ignoring the Kestrel), and a fleet able to take down any ship in human space. I know from experience it'd only take me another 2 hours or so to have a fleet that's mostly untouchable and able to capture a Skylark with a min/maxed Rano'erek .

I don't have a specific solution in mind, but this seems too fast. My frame of reference is the other Escape Velocity games, where iirc you were forced to engage with missions in order to open up access to more powerful ships and weapons. I'm not saying missions have to be the way that amassing power is limited, but that is one way to do it. It does feel a bit of a pity that you can have the most powerful fleet in the universe without ever playing a single mission.

If anyone's curious how I did it, I basically hung out in Aldhibain to get a fleet of Furies, Hawks, Clippers and Argosies, and then when I had enough firepower I went to Beta Lupi and Men to get myself Bastions, Leviathans and Falcons. I grabbed frag grenades (and later nerve gas) in Antares to maximise capture odds.

I'm sure there are other ways to do it as well, and somebody's probably done it faster - this wasn't an attempt at a speed run, nor a boast, just normal play in Human space to prove a point and set up this discussion. I'm curious what people think.

**Comment by Saugia** [comment]
Although it is closed for now, https://github.com/endless-sky/endless-sky/pull/8506 is the most recent thoughts to kind of slow this down. But yeah, the ease of accelerating the economy and combat capability of the player is something we'd like to address.

**Reply by brendanjones** [reply]
Making ship capture harder is certainly one way of approaching it.

After a quick brainstorm, a few other ideas for ways of slowing things down (I don't know which of these have already been discussed except for the one for which I've linked an issue):
- Lower the capture odds of the Kor Sestor and Kor Mereti ships, making it slower to amass a fleet of them.
- Increase crew costs, making a large fleet less economical.
- Add some sort of cost to owning Kor Sestor and Kor Mereti ships, so that you're limited in the same way that crew costs limit (to some extent) ownership of other ships.
- Lower the DPS of human weapons like in https://github.com/endless-sky/endless-sky/issues/9802
- Only open up the wormholes (to the Hai and Remnant) after completing missions for them. Especially the Hai one, because you can capture Unfettered Hai Shield Beetles with impunity, making them a super fast way to get a powerful fleet. Whereas the Remnant won't let you land if you start capturing their ships, meaning you at least have to take each captured ship out of Remnant territory to another planet, which is tedious. 
- Alternatively (or additionally?) to making wormhole access limited by missions, you could limit how easy it is to plunder and capture Unfettered Hai. This could be through changes to the ships themselves, or, say, by making normal Hai hostile towards you for attacking Unfettered Hai. I don't think this breaks lore (much?) given the Hai see the Unfettered as misguided brothers.
- Give Remnant ships better capture defense, to slow down their capture.
- Make Pug ships unplunderable, or harder to disable, or remove jump drives from Zibrukas, or (...?) because raiding Pug ships for Jump Drives in Deneb is a stupidly fast way to outfit your fleet with jump drives as soon as you've plundered one jump drive off of a Korath Exile ship.
- Lower the number of Korath Exile ships that proc in human space, making them a far slower source of T2 tech (especially jump drives).
- Lower the attack stats of frag grenades and nerve gas especially, which both make capturing ships with larger crew counts too easy.
- Give planets stronger defense fleets, because demanding tribute without losses is easy after about 2 hours of play time (for the weakest planetary fleets).

**Reply by Saugia** [reply]
Responding to these points - 
> Lower the capture odds of the Kor Sestor and Kor Mereti ships, making it slower to amass a fleet of them.

At the moment a large portion of the playerbase is already concerned with the random capture odds of these ships and there have been many requests to make it less random and there are many plug-ins that remove it entirely. Though, the PR I linked partially hints to the idea that the player would need more than just random chance to unlock said ships, some sort of tool or something, so maybe we lean towards that and not rng, but we'll see.

> Increase crew costs, making a large fleet less economical.

Like the above, there's already concern that crew costs are annoying to uphold, however like mentioned here the ease of economy otherwise makes it offset pretty easily. Rather than increase requirements, I think it would be better to look at decreasing efficiency. Could be a balance of both, though.

> Lower the DPS of human weapons like in https://github.com/endless-sky/endless-sky/issues/9802

Certainly. This PR is still a decently-large "maybe" while we figure out exactly what direction we want to to tug it in. It would help tangentially, but obviously wouldn't be a direct solution.

> Only open up the wormholes (to the Hai and Remnant) after completing missions for them. Especially the Hai one, because you can capture Unfettered Hai Shield Beetles with impunity, making them a super fast way to get a powerful fleet. Whereas the Remnant won't let you land if you start capturing their ships, making it impossible to bring ships out of Remnant space because they don't have a keystone to navigate the wormholes.

Having a sort of "directed story" through accessibility has been brought up before, but we don't want to explicitly direct the events of the game and lock areas because of it (i.e., only access the Hai after blank, then only the Remnant after Hai, etc..) The only real lock like this is the ownership of the JD, but we don't want to "section off" areas as unlockables.

> Alternatively (or additionally?) to making wormhole access limited by missions, you could limit how easy it is to plunder and capture Unfettered Hai. This could be through changes to the ships themselves, or, say, by making normal Hai hostile towards you for attacking Unfettered Hai. I don't think this breaks lore (much?) given the Hai see the Unfettered as misguided brothers.

This was already done once, actually. Before, Uhai Shield Beetles were pretty easy to find disabled in Wah Ki after a brief amount of time. In the last ~year, they were given the `coward` personality, so after they get low, they will leave the system rather than fight to disabled. There's been response to this change in that it has decreased their disabled count noticeably. So, on top of this, something like the linked PR above would stack on top of that.

> Make Pug ships unplunderable, or harder to disable, or remove jump drives from Zibrukas, or (...?) because raiding Pug ships for Jump Drives in Deneb is a stupidly fast way to outfit your fleet with jump drives as soon as you've plundered one jump drive off of a Korath Exile ship.

Could be something to discuss since nothing stops the player especially from stealing them from a bunch of small ships that get disabled. Then again, the player's ability to obtain ships and farm such a fleet to begin with is still affected by the linked PR.

> Lower the number of Korath Exile ships that proc in human space, making them a far slower source of T2 tech (especially jump drives).

Prior to Misam, Durax and Atik's existence , Korath ships appeared very rarely within Polaris, Sheratan and Almach. Though, even if you can take JDs from these ships, the systems they spawn in more are dangerous for the player with the Pirate presence, so it is normally difficult to have enough firepower to fight both sides and get away with it until you gather a strong enough force, going back to the ability to get the force, not necessarily the rarity of the Korath ships. However I will say that after watching for a few updates how those spawns are, they could be pulled back a little bit.

> Lower the attack stats of frag grenades and nerve gas especially, which both make capturing ships with larger crew counts too easy.

A lot of it is indeed Nerve Gas. I know nerfs have been suggested, but I don't recall any concrete decision or thoughts on it, so we could maybe discuss it, but to be a broken record, having new boarding requirements on top of them would still be considered first.

>Give planets stronger defense fleets, because demanding tribute without losses is easy after about half a day of play time.

Planetary tribute isn't a canon part of the game. It's more so a "mini game" for players who have completed the story. If the player somehow has obtained a massive fleet or combat capability before then, we should be taking a look more at how they obtained that rather than immediately buff other sources around that capability. (Same thing as above with nerfing undesirably strong efficiency vs. balancing around it.)

And just as a general thought - we'd definitely be willing at looking to adjust efficiency, but we wouldn't go as far as to completely remove the capability to do something. (For example, fleet gameplay is not ES's intended experience, but we wouldn't hard-limit fleets.)

**Reply by brendanjones** [reply]
Thanks for the context on what's previously been discussed (and hasn't), I appreciate it.

**Reply by Quantumshark** [reply]
> Lower the DPS of human weapons like in https://github.com/endless-sky/endless-sky/issues/9802

Implemented in #9851, if you're interested in trying it out.

**Reply by SultanRTG** [reply]
Nerve gas is semi nerfed with the illegal repercussions.

**Reply by brendanjones** [reply]
Nah, if you have anything like an Argosy or better then you can capture ships worth selling, meaning the fine isn’t much of a hindrance. It would be a hindrance if interference plating worked less well (i.e. you get fined more often), but honestly I wouldn’t make the $1 million fine mechanic harder on the player, it’s annoying enough as it is.

**Reply by brendanjones** [reply]
@Saugia 
> Lower the number of Korath Exile ships that proc in human space, making them a far slower source of T2 tech (especially jump drives).

> Prior to Misam, Durax and Atik's existence , Korath ships appeared very rarely within Polaris, Sheratan and Almach. Though, even if you can take JDs from these ships, the systems they spawn in more are dangerous for the player with the Pirate presence, so it is normally difficult to have enough firepower to fight both sides and get away with it until you gather a strong enough force, going back to the ability to get the force, not necessarily the rarity of the Korath ships. However I will say that after watching for a few updates how those spawns are, they could be pulled back a little bit.

I was actually paying attention to the proc rate when I was playing just now: Jumping into Misam I had four Palavrets proc in the space of less than a minute (approx). That's way too many, IMO. They shouldn't proc in every border system you jump into, they should be a rarity that you have to search around for.

> Like the above, there's already concern that crew costs are annoying to uphold, however like mentioned here the ease of economy otherwise makes it offset pretty easily. Rather than increase requirements, I think it would be better to look at decreasing efficiency. Could be a balance of both, though.

Huh, honestly I've never found them much of a hindrance except when making absolutely huge fleets of, say, Korath ships. I definitely wouldn't *lower* the crew cost, that's for sure.

>  This was already done once, actually. Before, Uhai Shield Beetles were pretty easy to find disabled in Wah Ki after a brief amount of time. In the last ~year, they were given the coward personality, so after they get low, they will leave the system rather than fight to disabled. There's been response to this change in that it has decreased their disabled count noticeably. So, on top of this, something like the linked PR above would stack on top of that.

Making them `coward` was a good change! They still get disabled a bunch just because the firefights in Wah Ki are intense, so I didn't find it difficult to grab a bunch of SB's quickly with my Falcon. Another potential change would be upping the crew count of SBs so they're not easily capturable with early game human ships (i.e. way harder for a Falcon without min/maxing to the absolute max). Not sure how that fits in with Hai lore, and it would have other knock on effects (some I can think of: fleets of SBs would be more expensive to maintain due to higher crew count, and having one would then make capturing other high crew-count ships (e.g. Korath) easier).

> A lot of it is indeed Nerve Gas. I know nerfs have been suggested, but I don't recall any concrete decision or thoughts on it, so we could maybe discuss it, but to be a broken record, having new boarding requirements on top of them would still be considered first.

It's of course fun to cap ships so if that's the aim then having nerve gas be so OP is fine. But otherwise it's weird to me that human tech is all T1, and then the human hand to hand combat outfit is stronger than H2H outfits from every single other race in the galaxy, with the exception of Heliarch Enforcer Confrontation Gear which is not quick or easy to get. And it's got over double the attack of the next best human outfit, frag grenades, so that's a huge jump. So yeah, having such an OP outfit available with minimal downside (the fines are easy to deal with) from the start of the game just doesn't seem right to me.

The attack rating of the best H2H outfit from each race: Hai: 0.7, Remnant: 1.2, Korath: 1.6, Pug: 2.3, Bundorea: 1.1, Gegno: 1.9, Heliarch: 3.1. The only one that's even close that you're able to get early game-ish is the Pug one, and that still requires a decent fleet equipped with jump drives, and a bit of grinding to capture Pug ships and take them back to an outfitter to strip their outfits, and you have to piss off the Pug.

So yeah, this might be beating a dead horse but I think drop nerve gas attack to at least 2.4 or 2.5. I'd even be happy to see it drop to 2.1 or 2.2. Then you have a real reason to consider the H2H outfits from other races (as in nerve gas still has the strongest H2H attack, but only by a fraction so the advantage is minimal when you take into account the illegality).

> Planetary tribute isn't a canon part of the game. It's more so a "mini game" for players who have completed the story. If the player somehow has obtained a massive fleet or combat capability before then, we should be taking a look more at how they obtained that rather than immediately buff other sources around that capability. (Same thing as above with nerfing undesirably strong efficiency vs. balancing around it.)

Yeah but right now it's just *so* easy. Blink-and-you-miss-it easy. I think you could make planetary fleets twice as big in number and twice as strong (stronger ships) and it'd still be easy.

**Reply by opusforlife2** [reply]
Just to nitpick a little: the best H2H attack weapon right now is Lunarium's Anti-Materiel Gun, at 3.4 attack.

**Comment by Hecter94** [comment]
You might want to check out #8506 as it's planned for eventual inclusion, and should help a lot with slowing down capturing in specific areas, such as making particularly powerful ships locked behind specific capturing outfits or requiring story progress to unlock the ability to capture some ships.

**Reply by Saugia** [reply]
Ninja'd.

**Comment by xX-Dillinger-Xx** [comment]
I agree that something should be done to slow down the ability to create huge fleets so quickly. 
However, there is an endless debate surrounding this. 
> fleet gameplay is not ES's intended experience

But the players want fleets and complain when the devs threaten to make it harder to do that. Many want to make it easier to capture even bigger fleets.
>..... been many requests to make it less random 

I personally believe a player should have to work at creating a fleet. Over coming a challenge is rewarding. It shouldn't just be handed over so easily. Vanilla ES should be the place for a challenging game play. If people want easy they can use the plugins, and will for sure. For some reason, I'll never understand, there are people that want maximum reward with little effort. I guess it's just a reflection of today's society. 

>Having a sort of "directed story" through accessibility has been brought up before, but we don't want to explicitly direct the events of the game and lock areas because of it (i.e., only access the Hai after blank, then only the Remnant after Hai, etc..) The only real lock like this is the ownership of the JD, but we don't want to "section off" areas as unlockables.

I understand the desire to keep the game open, but don't think of "blocks" as directing the path. Think of them as challenges to over come to proceed. Locking wormhole access isn't a truly bad idea if done right. I wouldn't lock it behind "completion" of storylines but lock behind something like smaller missions and achievements. Maybe, you can't see them until you gather the necessary information as to how to detect them (fine tuning your sensors)  and no one wants to share that information with a rookie pilot.

We already have one species that will "end your game" if you are caught with anything of theirs. Maybe, we could have more of something similar. After all, capturing ships is basically piracy. As an example, we can capture Republic ships, but for some reason they don't care. Maybe, governments can send "bounty hunter's or repo people" to try and recapture the stolen ships. We can except "bounty missions" so maybe other pilots do the same thing. That leads to the question, is it possible to have your flagship or escorts captured by pirates? I've had escorts boarded but nothing seems to happen, or I'm I imagining that?

**Reply by lumbar527** [reply]
> > fleet gameplay is not ES's intended experience
> But the players want fleets and complain when the devs threaten to make it harder to do that. Many want to make it easier to capture even bigger fleets.

Fleets can make up for a lack in power; in my opinion you need a fleet to complete FW or at least higher-tier alien ship.

**Reply by Saugia** [reply]
Certainly. Fleets can be seen as a way to make content easier if it proves too difficult. The campaign is entirely possible to do with some difficulty in a single ship, and pretty smoothly with just a few. If the player is having difficulty, having a fleet can be seen as a way to increase the ease of the game. Still though, our intent is not to cater to fleet gameplay or make it an easy alternative, we can and still will revisit content balance for solo play.

**Comment by yafflehk** [comment]
I think one of the ways to reduce this kind of play is to have more "Early Game" content. Even at the very beginning of the game the player is  talking about (and guided towards)  "getting something better soon" but if there was more stuff to do and goals to attain for a shuttle/star barge then people would stick with that for longer. I'm thinking about other licenses that unlock upgrades or trade routes, guilds, reputations or private companies that offer protection or advancement. Smaller upgrades that work with these ships at this level. Right now the game is interesting at the beginning, a small ship exploring the galaxy alone,  but as soon as you start amassing bigger ships and a fleet I find it gets very predictable. It would be nice if the player could think about buying a Heavy Shuttle as a cool upgrade and a real goal.

**Reply by yafflehk** [reply]
One thing that occurred to me after writing this was that upgrades should scale somehow. For example if I buy a fuel tank for a mule or a hauler it will give me 1 more jump, but if I buy the same tank for a small ship it should give me 2, or even 3. I don't know if the game engine can do that kind of thing but that kind of scalability would make a smaller ship attractive for longer and enhance the early game.

**Reply by MasterOfGrey** [reply]
> I don't know if the game engine can do that kind of thing but that kind of scalability would make a smaller ship attractive for longer and enhance the early game.

The ability to have drives use different amount of fuel for ship sizes exists: https://github.com/endless-sky/endless-sky/pull/5609
It is not used for any vanilla content though.

**Reply by Saugia** [reply]
Some examples like this are currently in the game. See the Coalition "Shield Refactor Module" as an example. It's been noted as a mechanic, though the general gist is that if we do add any sort of multipliers to values (whether they currently exist or not), it is supposed to be done minimally and hesitantly as it's a very hard thing to balance. So generally we don't plan on adding any universal changes that incorporate multipliers in outfits or ships, but it could be considered for specific cases, most likely alien use.

Nova had multipliers, but they made certain ships wildly outbalanced compared to others, and buffed others that did not have a means to be nerfed properly in return. Aside from ship by ship basis, multipliers suddenly require balancing across the entire board as something that might not seem that strong on a small ship could be broken on a large ship with tons of space, or vice versa. For example, a 10% requirement of something on a small ship is much more impactful than something on a large ship.

**Comment by brendanjones** [comment]
Okay this one takes the cake, I'm laughing here. I started a new pilot, and then in 17 minutes I had a Bastion. I literally captured a Bastion with a Shuttle haha. Didn't even have frag grenades, just laser rifles and 18 crew. And no, I also wasn't reloading the file until I got something good, this was with zero reloads. Yes, lucky, I know. So 17 minutes in I have a juicy warship and 5 million net worth (I captured some scrappers, sparrows, hawks, clippers and argosies in that time, too).

Yeah, the capturing mechanic needs a wee nerf haha.

<img width="299" alt="Screenshot 2024-04-13 at 00 01 30" src="https://github.com/endless-sky/endless-sky/assets/16049594/9de3adb3-5c7a-4c99-87fc-8443c161372d">

**Comment by CrazyIvan99** [comment]
I know I am late to this conversation, but a simple way to slow down progression would be to increase the crew requirements for warships. Even the smallest modern warships have compliments in the hundreds, the largest in the thousands. No civilian ship should ever carry enough crew to capture a warship.

**Reply by tibetiroka** [reply]
That wouldn't solve the issue of a warship capturing warships; in fact, that would make it even easier to capture freighters with warships.

While it may slow down progression in the very early game, it might make things a lot worse once you are able to afford even a small warship.

**Reply by CrazyIvan99** [reply]
I don't necessarily agree that it should be difficult for a heavy warship to capture a freighter. And if all heavy warships are running around with 2000 crew, they are going to beat eachother down to a skeleton crew in a capture attempt. 

The original question was how to slow down early game progression, which I think this would accomplish. Late game power is going to snowball regardless.

**Reply by brendanjones** [reply]
> The original question was how to slow down early game progression, which I think this would accomplish.

It probably sounded like I was focussed on the early game because I was talking about the early game in my example, but I do think capturing a fleet needs slowing down at all stages of the game.

**Comment by SpearDane** [comment]
This is definitely my biggest issue with balance. The fleet mechanic breaks the game in terms of combat and economy. Just snowball until your problems go away. 

I think if people love their fleets and complain about them being harder to gain/maintain or limiting the number of escorts.. or whatever work around they want.. they should be encouraged to use a mod that does that. That's where balance breaking mechanics belong.

**Reply by brendanjones** [reply]
Can’t agree enough with this.

**Comment by opusforlife2** [comment]
The easy fleet-building is somewhat countered by the dumb fleet AI. If you have a large fleet that you've spent tons of money on, then you need to put in more effort to make sure they stay alive too, because they will all happily kamikaze themselves in a burst of glory if they get the slightest chance to.

**Reply by ghost** [reply]
I don't think so. Yes, the AI is.. Well... A bit kamikaze, but at the end, everything pass for enjoying the stars from Mesuket and capture enough 349 or 512 to burn no matter wich system to ashes. It's only time and it can be done with a Starling/Penguin.

**Comment by ghost** [comment]
I am doing a plugin just about that. If you want, with a simple shuttle in a few weeks yo could be piloting a Falcon... with it's respective escorts, of course. 
There's another plugin doing something like that (Spacefarer) but i think than negates the boarding mechanics at all is not the way to do. Instead, a ship disabled, or plenty of holes, is very close to explode. A simple random mission that blows the ship (Or self-destruct) on board is easy to implement. 

On the same way, if you, as captain, don't understand a single word in Hai, Korath, or whatever the Coalition uses as language, is a joke you can board, kill, and piloting. A bit of logic, please.
Moreover... You disable a ship... board, and, while the enemy crey looks at you without do anything, you, and your completely anonimous crew, take the generator, jump drive, or whatever, and fly away. I can imagine the scene, with some humans entering the Palavret with tool boxes to dismantle a 160 tons generator to throw in the cargo bay. In space, sometimes between a battle, while 250 reptilian are "doing their things".

¿More? ¿There's a way to get in game how many fuel you have to replenish in human space... And pass the bill to the captain?. ¿A deuterium space-station refinery is mantained for free? ¿The mechanics who repair your hull?. 
This is what I don't know how to do. I beg for ideas, please.

I like the game so much, and i was playing it since years... But there's things that irch me.
The real solution, from me, pass for making hard a capture, and shows a real society where people get paid for their work. Or, if you want to speak in other way, making your way as a captain harder.

**Comment by yafflehk** [comment]
Couldn't the mechanic of hiring crew to capture ships be changed? For example, if you have captured more than n ships you get a bad reputation, not quite a pirate but a bit dodgy. This would mean that most spaceports would have the message "There are no crew for hire here" when you try to hire crew, maybe only pirate worlds would hire to you, but with a n% chance of a mutiny when you try to cap a ship if all your crew were hired at a pirate world?

**Comment by ieatfishes** [comment]
A few thoughts for crew numbers scaling up difficulty:

1) Every additional crew member costs more money. Similar to something like CivClicker (https://civclicker.sourceforge.net/civclicker/civclicker.html) where every time you create a worker the cost goes up. The first worker is 20 but if you have 10,000 people in your town already it costs maybe 120 per new worker. As you lose population, new workers are cheaper again. So it just scales up making larger ships and fleet cost more and more each ship (and crew) you add. In lore, this could be explained by the infrastructure and upkeep of a large worker force: larger lodging on planets, more specialized crew like medical, HR, specialists, etc.

2) A 'seasoned crew' mechanic. Similar to a ranking up system or seniority type of thing. If you have a crew member on the payroll longer, they become a seasoned crew or rank up or somehow are different from a fresh recruit. Maybe higher pay but also better capture odds, worth 1.5 of a crew requirement, etc. This could reward spending time in a ship longer, disincentivize hiring large fleets for single jobs, or give small increases in ship performance. If you buy a new ship, the crew starts fresh and has to rebuild that experience.

3) Your combat rating or crew lost in boardings negatively affects the expense to hire more crew. This can go away over time like a credit score. But for every crew you lose, you get a penalty for hiring new crew and/or, as mentioned by yafflehk, some places won't even have enough crew to hire. This made me think of the next idea of

4) Planets only have a certain amount of crew available. Want to hire 500 people for your fleet of shield beetles? Too bad, there are only 20 on this planet who are qualified to run a ship like that. Maybe have tiers of crew. Captains, specialists, deck hands, etc. with different salary amounts. You would need to hire enough for a ship BEFORE you can buy a ship and might need to shop around. And if you clear out a planet of their specialists, it might take a year to get back to that number.

Anyway, just some ideas I had. I definitely think tweaking crew usage would solve a lot of scaling issues rather than trying to limit ships by type, faction, etc. I also think parking ships should still cost you money one way or another. Even if not the full crew salary, there has to be upkeep, hangar space, maintenance, etc.
